try-it.md

```diff
@@ -113,10 +123,10 @@ $ sudo virsh list
 
 Each of the VMs (aside from the `minikube` management cluster VM) are
 represented by `BareMetalHost` objects in our management cluster. The yaml
-defition file used to create these host objects is in `bmhosts_crs.yaml`.
+definition file used to create these host objects is in `bmhosts_crs.yaml`.
 
 ```sh
-$ kubectl get baremetalhosts -n metal3
+$ kubectl get baremetalhosts -n metal3 -o wide
 NAME     STATUS   PROVISIONING STATUS   CONSUMER   BMC                         HARDWARE PROFILE   ONLINE   ERROR
 node-0   OK       ready                            ipmi://192.168.111.1:6230   unknown            true
 node-1   OK       ready                            ipmi://192.168.111.1:6231   unknown            true
```

Comment:
Print columns have changed recently, maybe it is worth to update them too in the same PR?
```suggestion
$ kubectl get baremetalhosts -n metal3 -o wide
NAME     STATUS   STATE   CONSUMER   BMC                         HARDWARE_PROFILE   ONLINE   ERROR
node-0   OK       ready                            ipmi://192.168.111.1:6230   unknown            true
node-1   OK       ready                            ipmi://192.168.111.1:6231   unknown            true
```

Reply:
Done in c24db1d, please take a look and I will squash it into the previous commit.

Reply:
@wgslr This looks good. Thanks.

Reply:
I will lgtm once you squash the commits.